modules/dnn/src/dnn.cpp

```diff
@@ -588,8 +535,7 @@ struct DataLayer : public Layer
 
     virtual bool supportBackend(int backendId) CV_OVERRIDE
     {
-        return backendId == DNN_BACKEND_OPENCV ||
-               (backendId == DNN_BACKEND_INFERENCE_ENGINE_NN_BUILDER_2019 && inputsData.size() == 1);
+        return backendId == DNN_BACKEND_OPENCV;
```

Comment:
We need to support this layer with nGraph as for me

Reply:
Ddo we need `inputData.size () == 1`condition?

Reply:
At least match DNN Builder implementation